git-squash-tags

```diff
@@ -8,16 +8,18 @@ if [ $# -eq 1 ]; then
 fi
 
 echo "Grabbing tags following pattern $tag_prefix..."
-tags=(`git log --reverse --pretty=%d --tags=$tag_prefix --no-walk | sed 's/ (tag: //' | sed 's/)//'`)
+tags=(`git log --reverse --pretty=%d --tags=$tag_prefix --no-walk |grep 'tag:' | sed 's/.*tag: //' | sed 's/)//'|sed 's/,.*//'`)
```

Comment:
I have run the script on branch called 1.5.x of git repo https://github.com/freeplane/freeplane.git .
`git log --reverse --pretty=%d --tags=$tag_prefix --no-walk` produced some lines with more than one tag like `git log --reverse --pretty=%d --tags=$tag_prefix --no-walk` . Also HEAD was indicated. I just fixed it to work on my computer.
(tag: beta-1.5.8-pre02, sf/1.5.x)